crypto/ct/ct_log.c

```diff
@@ -171,18 +171,25 @@ static int ctlog_store_load_log(const char *log_name, int log_name_len,
 
     /* log_name will be NULL for empty list entries */
     if (log_name == NULL)
-        return 1;
+        goto err;
 
     tmp = OPENSSL_strndup(log_name, log_name_len);
+    if (tmp == NULL)
+        return 1;
```

Comment:
Hmmm...a malloc failure, to me, seems to meet the criteria for "something very bad happens". So I really would return 0 in this case. I think my original comment just suggested tweaking the wording of the comment above - not turning everything to "return 1".